src/platform-includes/getting-started-install/react-native.mdx

```diff
@@ -8,49 +14,41 @@ npm install --save @sentry/react-native
 yarn add @sentry/react-native
 ```
 
-### Expo
-
-If you are using Expo, see [How to Add Sentry to Your Expo Project](https://docs.expo.io/guides/using-sentry/). This SDK works for both managed and bare projects.
-
-<Note>
-
-Make sure that the version of `@sentry/react-native` matches what `sentry-expo` depends on, [read more on our troubleshooting page](/platforms/react-native/troubleshooting/#expo).
+Run `@sentry/wizard`:
 
-</Note>
+```bash
+npx @sentry/wizard -i reactNative -p ios android
 
-### Linking
+cd ios
+pod install
+```
 
 <Note>
 
-If you are running a project with `react-native` prior to 0.60, you still need to call `react-native link`, otherwise you can skip this step as `react-native` version 0.60 and above does this automatically.
+For React Native 0.68 and older you will have to manually patch the Xcode project. Read more at the [Manual Configuration](/platforms/react-native/manual-setup/manual-setup/) page.
 
 </Note>
 
-Link the SDK to your native projects to enable native crash reporting.
+[Sentry Wizard](https://github.com/getsentry/sentry-wizard) will patch your project accordingly, though you can [setup manually](/platforms/react-native/manual-setup/manual-setup/) if you prefer. You need to do this only once, then the created files can go into your version control system.
 
```

Comment:
Thanks for all the suggestions. 

With the `can` here, I wanted to express that people can add there to their VSC but they don't have to.

Does `will` mean the same thing here?

Reply:
To give people more of a feeling of choice, I'd suggest being more explicit about it and saying something like: "If you want to add files to your VCS, you need to only add them once, they will then go into your version control system."

Otherwise, saying "can" instead of "will" makes it sounds like we're not sure where the files will go.

Reply:
I believe what Krystof is trying to say, is something along these lines: "You only need to patch the files once. Then you can add them to your version control system."

Reply:
```suggestion
[Sentry Wizard](https://github.com/getsentry/sentry-wizard) will patch your project accordingly, though you can [set up manually](/platforms/react-native/manual-setup/manual-setup/) if you prefer. You only need to patch the project once. Then you can add the patched files to your version control system.
```

@lizokm Would this work?